releasenotes/notes/add-option-to-collect-sql-metadata-9b22dd3f1fe30433.yaml

```
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    Add the option to collect SQL comments and commands during obfuscation.
```

```suggestion
    APM: Add the option to collect SQL comments and commands during obfuscation.
```
We need to add this prefix for trace-agent.